crates/pipeline-manager/src/db/operations/pipeline.rs

```diff
@@ -514,6 +530,7 @@ pub(crate) async fn update_pipeline(
     name: &Option<String>,
     description: &Option<String>,
     platform_version: &str,
+    mut bump_platform_version: bool,
```

Comment:
would be nice to update existing unit tests to account for this new logic

Reply:
ok, did my best.